limbo/config.py

```diff
@@ -0,0 +1,50 @@
+import os
+from backports import configparser as confparser
+
+
+class Config:
+    env_config = {}
+    file_config = None  # type: confparser.ConfigParser
+
+    def __init__(self):
+        self.load_env_config()
+        self.load_config_file()
+
+    def load_config_file(self):
+        self.file_config = confparser.ConfigParser()
+        config_file = self.env_config.get("config_location", "config.ini")
```

Comment:
Does this really need to be configurable? Couldn't we just say that the config file is named `config.ini`?

Reply:
It makes testing easier, to change between mock files. Also adds easier configuration.